lib/message_bus/backends/postgres.rb

```diff
@@ -131,20 +132,23 @@ def subscribe(channel)
           listener = Listener.new
           yield listener
 
-          conn = raw_pg_connection
-          conn.exec "LISTEN #{channel}"
+          @subscribe_connection = raw_pg_connection
```

Comment:
Maybe just `conn = @subscribe_connection = raw_pg_connection`, to avoid unnecessary instance variable access and reduce the scope of the changes.